site/survey.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="author" content="James-Konieczny">
+    <meta name="description" content="A personal website featuring a haiku, quote, and more.">
+    <title>James' Website</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"><!--adjust to different screen sizes-->
+    <link rel="stylesheet" href="styles/main.css">
+    <link rel="icon" type="image/png" href="/favicon-96x96.png" sizes="96x96" />
+    <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+    <link rel="shortcut icon" href="/favicon.ico" type="x-icon"/>
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
+    <link rel="manifest" href="/site.webmanifest" />
+    <link rel="icon" type="image/x-icon" href="favicon.ico">
+</head>
+<body>
+    <div class="width-wrap">
+        <header id="top">
+            <nav><ul>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="links.html">Links</a></li>
+                <li><a href="glossary.html">Glossary</a></li>
+            </ul></nav>
+        </header>
+        <main>
+            <div id="survey">
+                <h1>Survey</h1>
+                <form action="https://formspree.io/f/xvgrrwyd" method="post">
+                    <section>
+                        <p>
+                            <label for="name">
+                                <span>What is your name? </span>
+                                <strong><span aria-label="required">*</span></strong>
+                            </label>
```

Comment:
While not incorrect (generally wrapping your text in a parent element is good practice. But what was expected was something akin this for all of the labels (the abbreviations were only needed on the required fields)
```suggestion
<label for="name_field">What is your name? <abbr title="required" aria-label="required">*</abbr></label>
```